todo/src/components/Task.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export default function Task(props) {
+	return (
+		<div className="todocard">
+			<p className={props.completed === true ? 'strikethrough' : ''} onClick={() => props.toggleCompleted(props.id)}>
```

Comment:
This is a pretty long line. Here's a couple of things you can do to shorten it down:
```javascript
// take out the unnecessary comparison in your condition
<p className={props.completed ? 'strikethrough' : ''} onClick={() => props.toggleCompleted(props.id)}>
```
```javascript
// destructure
<p className={completed ? 'strikethrough' : ''} onClick={() => toggleCompleted(id)}>
```
```javascript
// or format on multiple lines
<p
  className={props.completed ? 'strikethrough' : ''} 
  onClick={() => props.toggleCompleted(props.id)}
>
```